config.js

```diff
@@ -0,0 +1,57 @@
+const NUM_ACTIONS = 5000;
+const TOTAL_POINTS = 100000;
+const POINTS_PER_ACTION = TOTAL_POINTS / NUM_ACTIONS;
+
+var configs = {
+	"namespace": "_", 		// Namespace to be used on OpenWhisk
```

Comment:
Isn't it namespace used by OpenWhisk on OpenShift?

Reply:
Oh yes, I'll make this more clear.